test/go/iterative_verifier_test.go

```diff
@@ -1,477 +0,0 @@
-package test
-
-import (
-	"database/sql"
-	"fmt"
-	"sort"
-	"testing"
-	"time"
-
-	"github.com/Shopify/ghostferry"
-	"github.com/Shopify/ghostferry/testhelpers"
-	"github.com/siddontang/go-mysql/schema"
-	"github.com/stretchr/testify/suite"
-)
-
-type IterativeVerifierTestSuite struct {
-	*testhelpers.GhostferryUnitTestSuite
-
-	verifier *ghostferry.IterativeVerifier
-	db       *sql.DB
-	table    *ghostferry.TableSchema
-}
-
-func (t *IterativeVerifierTestSuite) SetupTest() {
-	t.GhostferryUnitTestSuite.SetupTest()
-	t.SeedSourceDB(0)
-	t.SeedTargetDB(0)
-
-	tableCompressions := make(ghostferry.TableColumnCompressionConfig)
-	tableCompressions[testhelpers.TestCompressedTable1Name] = make(map[string]string)
-	tableCompressions[testhelpers.TestCompressedTable1Name][testhelpers.TestCompressedColumn1Name] = ghostferry.CompressionSnappy
-
-	compressionVerifier, err := ghostferry.NewCompressionVerifier(tableCompressions)
-	if err != nil {
-		t.FailNow(err.Error())
-	}
-
-	t.verifier = &ghostferry.IterativeVerifier{
-		CompressionVerifier: compressionVerifier,
-		CursorConfig: &ghostferry.CursorConfig{
-			DB:          t.Ferry.SourceDB,
-			BatchSize:   t.Ferry.Config.DataIterationBatchSize,
-			ReadRetries: t.Ferry.Config.DBReadRetries,
-		},
-		BinlogStreamer: t.Ferry.BinlogStreamer,
-		SourceDB:       t.Ferry.SourceDB,
-		TargetDB:       t.Ferry.TargetDB,
-
-		Concurrency: 1,
-	}
-
-	t.db = t.Ferry.SourceDB
-	t.reloadTables()
-
-	err = t.verifier.Initialize()
-	testhelpers.PanicIfError(err)
-}
-
-func (t *IterativeVerifierTestSuite) TearDownTest() {
-	t.GhostferryUnitTestSuite.TearDownTest()
-}
-
-func (t *IterativeVerifierTestSuite) TestNothingToVerify() {
-	err := t.verifier.VerifyBeforeCutover()
-	t.Require().Nil(err)
-
-	result, err := t.verifier.VerifyDuringCutover()
-	t.Require().Nil(err)
-	t.Require().True(result.DataCorrect)
-	t.Require().Equal("", result.Message)
-}
-
-func (t *IterativeVerifierTestSuite) TestVerifyOnceWithIgnoredColumns() {
```

Comment:
Not applicable as VerifyOnce is no longer needed as DataIterator directly calls InlineVerifier as the deltacopier runs. IgnoredColumns is covered by:

https://github.com/Shopify/ghostferry/blob/b8147c5ff8653099f6c5e24128aa6fd41ff401e1/test/integration/inline_verifier_test.rb#L80-L113